runners/apex/src/main/java/org/apache/beam/runners/apex/translation/operators/ApexTimerInternals.java

```diff
@@ -63,31 +68,37 @@ public void setContext(K key, Coder<K> keyCoder, Instant inputWatermark) {
     this.currentInputWatermark = inputWatermark;
   }
 
+  @VisibleForTesting
+  protected TimerSet getTimerSet(TimeDomain domain) {
```

Comment:
It is required for the unit test. It could be package scope, but it cannot be private.